test/crypto_sign/functest.c

```diff
@@ -28,6 +28,10 @@
 
 #define MLEN 1024
 
+#ifdef PQCLEAN_USE_SIGN_CTX
+#define PQCLEAN_CTXLEN 14
```

Comment:
Are there any limits on this context? Would it make sense to set it to a silly number like 10000, to catch out assumptions?